helm_resource/README.md

```diff
@@ -61,6 +62,8 @@ chart. Must be the same length as `image_deps`.  There are two common patterns.
 
 `flags`: Additional flags to pass to `helm install` (e.g., `['--set', 'key=value']`)
 
+`global_flags`: Additional flags to pass to all helm invocations (e.g., `['--namespace', 'app']`)
```

Comment:
are there any flags that you need beside `--namespace`? `app` isn't a flag...

feels like once you get into the business of parsing the flags (which is going to be tricky and error-prone), we might as well have a dedicated `namespace` argument

Reply:
this was to match up with the global flags section as shown`helm install --help` (and other helm commands)
```
...
Global Flags:
      --debug                       enable verbose output
      --kube-apiserver string       the address and the port for the Kubernetes API server
      --kube-as-group stringArray   group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --kube-as-user string         username to impersonate for the operation
      --kube-ca-file string         the certificate authority file for the Kubernetes API server connection
      --kube-context string         name of the kubeconfig context to use
      --kube-token string           bearer token used for authentication
      --kubeconfig string           path to the kubeconfig file
  -n, --namespace string            namespace scope for this request
      --registry-config string      path to the registry config file (default "/Users/markingram/Library/Preferences/helm/registry.json")
      --repository-cache string     path to the file containing cached repository indexes (default "/Users/markingram/Library/Caches/helm/repository")
      --repository-config string    path to the file containing repository names and URLs (default "/Users/markingram/Library/Preferences/helm/repositories.yaml")
```


my usecase is for namespace though, so happy to switch

Reply:
PR uses dedicated `namespace` argument now.